I1283 and i1324 cleaning creating repairing devfile section (#841)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-from-a-feature-branch-of-a-git-repository.adoc

// configuring-a-workspace-using-a-devfile

[id="creating-a-workspace-from-a-feature-branch-of-a-git-repository_{context}"]
= Creating a workspace from a feature branch of a Git repository

> Che 7 workspace can be created by pointing to devfile which is stored in a Git source repository, on the feature branch of the user’s choice. The Che 7 instance then uses the discovered devfile to build a workspace.

Reviewer: ```suggestion
A Che workspace can be created by pointing to devfile that is stored in a Git source repository on a feature branch of the user’s choice. The Che instance then uses the discovered devfile to build a workspace.
```

.Prerequisites

> * A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che 'quick-starts'].

Reviewer: ```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

> *  The `devfile.yaml` file in the root folder of  Git repository on specific branch of user choice available over HTTPS. See link:{site-baseurl}che-7//making-a-workspace-portable-using-a-devfile/[Making a workspace portable using a Devfile] for detailed information about creating and using devfiles.

Reviewer: ```suggestion
*  The `devfile.yaml` file in the root folder of a Git repository on a specific branch of the user's choice available over HTTPS. See link:{site-baseurl}che-7//making-a-workspace-portable-using-a-devfile/[Making a workspace portable using a devfile] for detailed information about creating and using devfiles.
```

.Procedure

> * Execute the workspace by opening the following URL:

Reviewer: ```suggestion

Execute the workspace by opening the following URL: `++https://__<CheHost>__/f?url=__<GitHubBranch>__++`.
```

> +

Reviewer: ```suggestion
```

> [subs="+quotes"]

Reviewer: ```suggestion
```

> ----

Reviewer: ```suggestion
```

> https://__<your-che-host>__/f?url=<yourgithubbranch>

Reviewer: ```suggestion
```

> ----

Reviewer: ```suggestion
```

> +

Reviewer: ```suggestion

```

> Example:

Reviewer: ```suggestion
.Example
```

> +

Reviewer: ```suggestion

```

> Use following URL format to open an experimental link:https://github.com/quarkusio/quarkus-quickstarts[quarkus-quickstarts] branch, hosted on `che.openshift.io`

Reviewer: ```suggestion
Use following URL format to open an experimental link:https://github.com/quarkusio/quarkus-quickstarts[quarkus-quickstarts] branch hosted on link:https://che.openshift.io[che.openshift.io].
```

> +

Reviewer: ```suggestion

```
